src/Nethermind/Chains/holesky.json

```diff
@@ -0,0 +1,1061 @@
+{
+  "name": "Holesky Testnet",
+  "dataDir": "holesky",
+  "engine": {
+    "Ethash": {}
+  },
+  "params": {
+    "accountStartNonce": "0x0",
+    "chainID": "0x4268",
+    "networkID": "0x4268",
+    "eip140Transition": "0x0",
+    "eip145Transition": "0x0",
+    "eip150Transition": "0x0",
+    "eip155Transition": "0x0",
+    "eip160Transition": "0x0",
+    "eip161abcTransition": "0x0",
+    "eip161dTransition": "0x0",
+    "eip211Transition": "0x0",
+    "eip214Transition": "0x0",
+    "eip658Transition": "0x0",
+    "eip1014Transition": "0x0",
+    "eip1052Transition": "0x0",
+    "eip1283Transition": "0x0",
+    "eip1283DisableTransition": "0x0",
+    "eip152Transition": "0x0",
+    "eip1108Transition": "0x0",
+    "eip1344Transition": "0x0",
+    "eip1884Transition": "0x0",
+    "eip2028Transition": "0x0",
+    "eip2200Transition": "0x0",
+    "eip2565Transition": "0x0",
+    "eip2929Transition": "0x0",
+    "eip2930Transition": "0x0",
+    "eip1559Transition": "0x0",
+    "eip3198Transition": "0x0",
+    "eip3529Transition": "0x0",
+    "eip3541Transition": "0x0",
+    "eip3651TransitionTimestamp": "0x65047260",
+    "eip3855TransitionTimestamp": "0x65047260",
+    "eip3860TransitionTimestamp": "0x65047260",
+    "eip4895TransitionTimestamp": "0x65047260",
+    "eip4844TransitionTimestamp": "0x77359400",
+    "eip4788TransitionTimestamp": "0x77359400",
+    "eip1153TransitionTimestamp": "0x77359400",
+    "eip5656TransitionTimestamp": "0x77359400",
+    "eip6780TransitionTimestamp": "0x77359400",
+    "terminalTotalDifficulty": "0x0",
+    "gasLimitBoundDivisor": "0x400",
+    "maxCodeSize": "0x6000",
+    "maxCodeSizeTransition": "0x0",
+    "maximumExtraDataSize": "0xffff",
+    "minGasLimit": "0x1388",
+    "registrar": "0x0000000000000000000000000000000000000000",
+    "MergeForkIdTransition": "0x0"
+  },
+  "genesis": {
+    "seal": {
+      "ethereum": {
+        "nonce": "0x1234",
+        "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+      }
+    },
+    "difficulty": "0x01",
+    "author": "0x0000000000000000000000000000000000000000",
+    "extraData": "0x686f77206d7563682069732074686520666973683f",
+    "gasLimit": "0x17D7840",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "timestamp": "0x65046234"
+  },
+  "nodes": [
+    "enode://ac906289e4b7f12df423d654c5a962b6ebe5b3a74cc9e06292a85221f9a64a6f1cfdd6b714ed6dacef51578f92b34c60ee91e9ede9c7f8fadc4d347326d95e2b@146.190.13.128:30303",
```

Comment:
probably it would be good to add more bootnodes once people will start their nodes.

Reply:
Sure, will check with dvops later once the network starts